### DISCUSSION TITLE
Are consumption fields available?

### DISCUSSION THREAD

**Discussion by adidoes** [tijsverkoyen/HomeAssistant-FusionSolar#133] [Q&A]
<img width="677" alt="image" src="https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/1117487/d3ae529e-707d-4518-a82d-47bba4fcb68b">

Fusion solar shows these stats for consumed, fed to grid, from PV etc.

Are these available from the plugin?

**Comment by tijsverkoyen** [comment]
In Kiosk mode the data that is available is very limited. 

With the Northbound / Open API method there is a lot of data available. This depends on the devices in your installation. I would suggest to ask your installer Northbound API credentials so you can check for yourself which data is available.

Also check https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/issues/94, this screenshot looks familiar. If I remember correctly not all numbers are available through the API, as some of them are calculations.

**Comment by majo551** [comment]
I have the same question, I checked my installation and I have all  the consumption allowed to be exported in the Northbound Account . However the HA integration does not provide the  Current Day nor Current Month Consumption , only Yearly and Lifetime one. 
![image](https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/assets/13347592/2e1d2e33-5894-4175-8cc6-f64cf9439895)
Actually for The Current Day and Month section, only very limited values integrated. Is is possible to include at least the  Daily and Monthly values for the Consumption?

**Comment by tijsverkoyen** [comment]
Feel free to share your credential with me (fusionsolar_ha [at] verkoyen [dot] eu) and I will check again if the information is available. But last time I checked it was not.
